src/packages/cli/src/Init.ts

```diff
@@ -275,11 +275,16 @@ export class Init implements Command {
       }
     }
 
+    const gitignorePath = path.join(outputDir, ".gitignore");
     try {
-      fs.writeFileSync(
-      path.join(outputDir, ".gitignore"),
-      defaultGitIgnore()
-      );
+      if (!fs.existsSync(gitignorePath)) {
+        fs.writeFileSync(
+          gitignorePath,
+          defaultGitIgnore()
+        );
+      } else {
+        warning = `${chalk.yellow('warn')} You already have a .gitignore. Don't forget to exclude .env`
```

Comment:
Add a new line `\n` before this so that this doesn't cram with other warnings.

Reply:
Since warning is a string this assignment here will overwrite a potential previous warning.
We need to find a way to preserve the previous warning.
Maybe switching `let warning` to `let warnings: string[]` and then printing `warnings.join('\n')` will be a good option here.